### ISSUE TITLE
Missing / mis-aligned interactions in RHS uploads

### ISSUE THREAD

**Issue by mwinokan** [m2ms/fragalysis-frontend#1409]
For the CHIKV_Mac target in production we have noticed that interactions are missing or mis-aligned for the RHS compounds

[snapshot](https://fragalysis.diamond.ac.uk/viewer/react/projects/112/107)

**Comment by mwinokan** [comment]
@boriskovar-m2ms @kaliif is it obvious to you what might be going wrong? This is for RHS uploads that do not have custom PDB's uploaded

**Comment by mwinokan** [comment]
<img width="503" alt="Screenshot 2024-04-11 at 09 18 53" src="https://github.com/m2ms/fragalysis-frontend/assets/36866506/e2c894a3-7d07-4010-8d85-65e95cd6f2f5">

This is a screenshot from the [snapshot](https://fragalysis.diamond.ac.uk/viewer/react/projects/112/107) I posted above. @boriskovar-m2ms can you see that none of the interaction 'dotted lines' are reaching the ligand. 

Do we have any control over how the interactions are generated? It's the NGL viewer that computes them right? Perhaps because the RHS uploads currently don't have a custom PDB the interactions are generated only from the APO (ligand-less) LHS reference.

**Comment by boriskovar-m2ms** [comment]
@mwinokan yes it's handled by NGL view and I'm just supplying an URL to the ```pdb``` file to be used. When RHS computed compound is referencing LHS observation then ```apo_file``` is used to render interactions.

![image](https://github.com/m2ms/fragalysis-frontend/assets/16878338/9b52d21e-5d74-40b9-8ca3-f212fc8a80a8)

**Comment by mwinokan** [comment]
Thank you @boriskovar-m2ms, so totally expected behaviour and will be fixed with custom PDB uploads #1348?

**Comment by mwinokan** [comment]
@phraenquex says we need to revisit these interactions on the whole, as the NGL computed ones are potentially misleading.
